raiden/encoding/messages.py

```diff
@@ -257,7 +242,7 @@ def wrap_and_validate(data):
 
     try:
         publickey = recover_publickey(message_data, message_signature)
-    except Exception:
+    except:  # pylint: disable=bare-except
```

Comment:
Not all exceptions are based on `Exception`. Anyways, this is wrong and we shouldn't use bare-excepts, I just didn't take the time to check what exceptions are raised by `recover_publickey`.

Specifically: This will prevent a greenlet from being killed.